src/lib.rs

```diff
@@ -361,6 +362,39 @@ impl ParserConfig {
         request.parse_with_config_and_uninit_headers(buf, self, headers)
     }
 
+    /// Sets whether invalid header lines should be silently ignored in responses.
+    ///
+    /// This mimicks the behaviour of major browsers. You probably don't want this.
+    /// You should only want this if you are implementing a proxy whose main
+    /// purpose is to sit in front of browsers whose users access arbitrary content
+    /// which may be malformed, and they expect everything that works without
+    /// the proxy to keep working with the proxy.
+    ///
+    /// This option will prevent `ParserConfig::parse_response` from returning
+    /// an error encountered when parsing a header, except if the error was caused
+    /// by the character NUL (ASCII code 0), as Chrome specifically always reject
```

Comment:
Should also mention `\r` here?

Reply:
Not exactly here, but I can mention it in a separate sentence yeah.

I reject `\r` because both Firefox and Chrome accept it, except in a different way (Firefox sees a plain old header value char, Chrome sees a header separator).

Reply:
Fun! I guess we can wait to find out whether someone will have a need for the FF or Chrome-specific behaviors. Maybe those could be extra flags in the future? Rejecting seems like the right call for now.